code/controllers/subsystem/runechat.dm

```diff
@@ -1,3 +1,112 @@
 TIMER_SUBSYSTEM_DEF(runechat)
 	name = "Runechat"
 	priority = FIRE_PRIORITY_RUNECHAT
+	/// List of most characters in the font. DO NOT CHANGE IT. ESPECIALLY VAREDIT.
+	var/list/letters = list(
+		//Special case, measuring " " returns 0 since it's empty string
+		" " = 2,
+		"." = null,
+		"," = null,
+		"?" = null,
+		"!" = null,
+		"\"" = null,
+		"/" = null,
+		"$" = null,
+		"(" = null,
+		")" = null,
+		"@" = null,
+		"=" = null,
+		":" = null,
+		"'" = null,
+		";" = null,
+		"+" = null,
+		"-" = null,
+		"\\" = null,
+		"<" = null,
+		">" = null,
+		"&" = null,
+		"*" = null,
+		"%" = null,
+		"^" = null,
+		"{" = null,
+		"}" = null,
+		"|" = null,
+		"~" = null,
+		"`" = null,
+		"A" = null,
+		"B" = null,
+		"C" = null,
+		"D" = null,
+		"E" = null,
+		"F" = null,
+		"G" = null,
+		"H" = null,
+		"I" = null,
+		"J" = null,
+		"K" = null,
+		"L" = null,
+		"M" = null,
+		"N" = null,
+		"O" = null,
+		"P" = null,
+		"Q" = null,
+		"R" = null,
+		"S" = null,
+		"T" = null,
+		"U" = null,
+		"V" = null,
+		"W" = null,
+		"X" = null,
+		"Y" = null,
+		"Z" = null,
+		"a" = null,
+		"b" = null,
+		"c" = null,
+		"d" = null,
+		"e" = null,
+		"f" = null,
+		"g" = null,
+		"h" = null,
+		"i" = null,
+		"j" = null,
+		"k" = null,
+		"l" = null,
+		"m" = null,
+		"n" = null,
+		"o" = null,
+		"p" = null,
+		"q" = null,
+		"r" = null,
+		"s" = null,
+		"t" = null,
+		"u" = null,
+		"v" = null,
+		"w" = null,
+		"x" = null,
+		"y" = null,
+		"z" = null,
+		MAX_CHAR_WIDTH = 0
```

Comment:
why is this a list entry and not just a var on the subsystem

Reply:
No reason in particular. Is there any difference though?

Reply:
It seems pointlessly unwieldy, and kinda uh, mixes the meaning of the list

Reply:
Addressed